### PR TITLE
change notify

### PR DESCRIPTION
**Pull request by ValuesFeng**
see code

### CODE REVIEW

**Comment by MarkoMilos** [comment]
Thanks, will look into this over the weekend and probably include it in the next release.

File: paginate/src/main/java/com/paginate/recycler/WrapperAdapter.java [hunk 1]
```diff
@@ -66,7 +66,11 @@ boolean isDisplayLoadingRow() {
     void displayLoadingRow(boolean displayLoadingRow) {
         if (this.displayLoadingRow != displayLoadingRow) {
             this.displayLoadingRow = displayLoadingRow;
-            notifyDataSetChanged();
+            if(displayLoadingRow){
+                notifyItemInserted(wrappedAdapter.getItemCount());
```
Comment (MarkoMilos) [inline paginate/src/main/java/com/paginate/recycler/WrapperAdapter.java]: This actually doesn't need to be true. Most of the times items will be added at the end of the list but it doesn't need to be the case. With this line you are saying that items are always inserted at the end of the list.

File: paginate/src/main/java/com/paginate/abslistview/WrapperAdapter.java [hunk 2]
```diff
@@ -67,6 +67,7 @@ public View getView(int position, View convertView, ViewGroup parent) {
 
     void displayLoadingRow(boolean displayLoadingRow) {
         this.displayLoadingRow = displayLoadingRow;
+        wrappedAdapter.notifyDataSetChanged();
```
Comment (MarkoMilos) [inline paginate/src/main/java/com/paginate/abslistview/WrapperAdapter.java]: I was looking around and found out that this is not the intention. In what case would you need to call notifyDataSetChanged upon wrapped adapter directly?

Comment (ValuesFeng) [inline paginate/src/main/java/com/paginate/abslistview/WrapperAdapter.java]: I need to call pagination.setHasMoreDataToLoad(boolean) whether the list data is null.
adapter always call onCreateViewHolder() and onBindViewHolder() while I call that.

**Comment by MarkoMilos** [comment]
This has been implemented as a part of the latest release (check out release 1.0.0)

I'm closing this MR.

**Pull request closed without merging**